Formula/nvc.rb

```diff
@@ -0,0 +1,140 @@
+  end
+
+  depends_on "llvm" => :build
+  depends_on "check" => :build
+
+
+  ieee_url="http://standards.ieee.org/downloads/1076/1076.2-1996"
+  vital_url="http://svn.gna.org/svn/ghdl/trunk/libraries/vital2000"
+  
+  resource "math_complex-body" do
+    url "#{ieee_url}/math_complex-body.vhdl"
+    sha256 "9dbdfaeb790f7afea854d52b84d20406717a6cff897ccc7720f10df53581c597"
+  end
+
+  resource "math_complex" do
+    url "#{ieee_url}/math_complex.vhdl"
+    sha256 "118a2c237befb730c7de09c2875a9f0cc3894e53d1d4c0c1c2779b8bc6dd0a31"
+  end
+  
+  resource "math_real-body" do
+    url "#{ieee_url}/math_real-body.vhdl"
+    sha256 "384516ee1ec0488a2e765c1b3cd6ab2212361f30195a46b6e044a408db4ba99a"
+  end
+  
+  resource "math_real" do
+    url "#{ieee_url}/math_real.vhdl"
+    sha256 "c09a96f905221da58f24c922ebdac886f69c2e80f102b6f29bb33c6bd9bd4c76"
+  end
+  
+  resource "numeric_bit-body" do
+    url "#{ieee_url}/numeric_bit-body.vhdl"
+    sha256 "5e0734c0b4eb70d7e6c776b6d639c526eff770eb8177f94d510678bc76c5b1c3"
+  end
+  
+  resource "numeric_bit" do
+    url "#{ieee_url}/numeric_bit.vhdl"
+    sha256 "3f4dfa268a0286eb24f89f4c1fe6d6122424acde93da0369fde53cce3d7e2b94"
+  end
+  
+  resource "numeric_std-body" do
+    url "#{ieee_url}/numeric_std-body.vhdl"
+    sha256 "739b4a68a27629b6e82b4e83a0f2cbd6127a70fe7b0ffd9e513d148bab5064ad"
+  end
+  
+  resource "numeric_std" do
+    url "#{ieee_url}/numeric_std.vhdl"
+    sha256 "1736e832e2d1ac2e94103f8199f0a83fd6070edf9fe5c1600c16189de6dd2672"
+  end
+  
+  resource "std_logic_1164-body" do
+    url "#{ieee_url}/std_logic_1164-body.vhdl"
+    sha256 "10f391f90869db4a05c2087d434d270b24fa8427218b64b68a827e270c47acc1"
+  end
+  
+  resource "std_logic_1164" do
+    url "#{ieee_url}/std_logic_1164.vhdl"
+    sha256 "c6359df78524b69052c8a6f8c485effbedb152b2bb5eecd0d38043b8013658c3"
+  end
+
+  # svn here: 
+  resource "timing_p" do
+    url "#{vital_url}/timing_p.vhdl", :using => :curl
+    sha256 "325f57aac8326d685bceecd4f3b05bd45b944cda07eaa7d395ab589e385e14ad"
+  end
+  
+  resource "prmtvs_p" do
+    url "#{vital_url}/prmtvs_p.vhdl", :using => :curl
+    sha256 "69f5f97a439a172b7b516314d28582aa28b8156fc9d826d096ed1e5b6aceb400"
+  end
+  
+  resource "memory_p" do
+    url "#{vital_url}/memory_p.vhdl", :using => :curl
+    sha256 "97c652ecc94a9f7eab13be8913776e2a98764f471e5a10dd6b9624eef8c24c11"
+  end
+  
+  resource "timing_b" do
+    url "#{vital_url}/timing_b.vhdl", :using => :curl
+    sha256 "e0981c345ff83423f1328d9739de652fa58bbc4a30004661e43e152eeb6ebdd7"
+  end
+  
+  resource "prmtvs_b" do
+    url "#{vital_url}/prmtvs_b.vhdl", :using => :curl
+    sha256 "d06dd5f5d5c4f8b19770db8e06b6302f87cb8e30e027feef201cc780152e3b8b"
+  end
+  
+  resource "memory_b" do
+    url "#{vital_url}/memory_b.vhdl", :using => :curl
+    sha256 "03878edc834050f67ec5d3e0b49bef883065dc429700a174e93850274c63e458"
+  end
+  
+  def install
+    args = [
+      "--with-llvm=/usr/local/opt/llvm/bin/llvm-config",
```

Comment:
Are you sure this is needed? It isn't autodetected by the buildsystem? Need to use `Formula["llvm"].bin` instead of hard-coding `/usr/local` if it's needed.